pages/storage_and_backup/object_storage/s3_encrypt_your_objects_with_sse_c/guide.fr-fr.md

```diff
@@ -148,7 +165,251 @@ Les URL pré-signées, qui peuvent être utilisées pour des opérations telles
 > [!primary]
 >
 > Vous pouvez donc utiliser l'URL pré-signée pour les objets SSE-C uniquement par programmation, car en plus de l'URL pré-signée, vous devez également inclure des en-têtes HTTP spécifiques aux objets SSE-C.
->
+
+### SSE-S3 - Chiffrement côté serveur avec clés gérées par OVHcloud
+
+L'utilisation du chiffrement côté serveur avec des clés gérées par OVHcloud (SSE-S3) permet de protéger vos données stockées sur OVHcloud en les chiffrant automatiquement au repos. SSE-S3 utilise des clés gérées et protégées par OVHcloud, éliminant ainsi le besoin pour l'utilisateur de gérer manuellement ces clés de chiffrement.
+
+#### Avantages
+
+- **Gestion des Clés Simplifiée**
+
+Avec OVHcloud prenant en charge la gestion sécurisée des clés de chiffrement, les utilisateurs bénéficient d'une gestion des clés simplifiée. Cette approche élimine les complexités liées à la rotation des clés tout en maintenant un niveau élevé de sécurité pour les données. Elle permet de concilier sécurité et efficacité opérationnelle, en éliminant la charge administrative de la gestion manuelle des clés de chiffrement.
+
+- **Sécurité Renforcée**
+
+Nous employons une stratégie de chiffrement avancée pour offrir une protection maximale de vos données. Chaque bucket bénéficie d'une clé unique, et pour chaque objet stocké, une clé de chiffrement distincte est générée. Cette clé est obtenue en combinant la clé unique du bucket avec un sel aléatoire, ce qui assure que chaque objet est chiffré avec sa propre clé. Cette méthode de dérivation de clés limite le risque associé à l'exposition d'une clé unique et garantit une sécurité renforcée pour vos données.
```

Comment:
`avec un sel aléatoire`
wrong translation?

Reply:
The term "sel aléatoire" in the context of encryption refers to the practice of adding random data when generating an encryption key, to ensure that each key is unique, even if the main key (or the unique key of the bucket) is reused.